org.freedesktop.Sdk.Extension.llvm12.json

```diff
@@ -137,7 +137,8 @@
         },
         "env": {
           "CC": "clang",
-          "CXX": "clang++"
+          "CXX": "clang++",
+          "LDFLAGS": "-Wl,--disable-new-dtags"
```

Comment:
This will override the default `LDFLAGS` which we may not want here so perhaps  use `ldflags` manifest option which will append to defaults.

Reply:
Done